src/Elastic.Markdown/Exporters/Elasticsearch/ElasticsearchExporter.cs

```diff
@@ -243,17 +246,24 @@ protected static string CreateMapping(string? inferenceId) =>
 		      },
 		      "stripped_body": {
 		        "type": "text",
-		        "search_analyzer": "highlight_analyzer",
```

Comment:
We need highlight_analyzer here so that the highlighted text comes in a certain way

Reply:
We already specify this highlighter during query time: 

https://github.com/elastic/docs-builder/blob/293a9434377c6d7eb7bf1d2fb2c4f5171dcbd90b/src/api/Elastic.Documentation.Api.Infrastructure/Adapters/Search/ElasticsearchGateway.cs#L150

This overrides the one in the mapping. `synonym_analyzer` seems like a good default.